draft-ietf-jsonpath-base.md

```diff
@@ -176,30 +182,31 @@ Name:
 : The name in a name/value pair constituting a member.  (Also known as
   "key", "tag", or "label".)
 
+Array:
+: A JSON array as defined in {{-json}}.
+  Never used in its generic sense, e.g., for programming language objects.
+
 Element:
 : An item in an array.  (Also used with a distinct meaning in XML
   context for XML elements.)
 
+Index:
+: An unsigned number that identifies a specific element in an array.
```

Comment:
Negative indices are also used in the spec.

Reply:
OK, we'll need to differentiate between the index as in number of element and the index as in expression value supplied to indexing.  That bothered me already about the existing text.

Reply:
Much better now ... Index definition is the real location and might not be confused with slice expression syntax.

Reply:
`FUNCTION Normalize(i)` should have `len` consistently as second argument, i.e. `FUNCTION Normalize(i,len)` ...

Reply:
Good catch; I have put the fix for Normalize into a separate PR #73